src/cursor/find_cursor.ts

```diff
@@ -353,6 +353,18 @@ export class FindCursor extends AbstractCursor {
     return this;
   }
 
+  /**
+   * Allows disk use for blocking sort operations exceeding 100MB memory.
+   */
+  allowDiskUse(): this {
```

Comment:
FWIW `allowDiskUse` is also supported on the aggregation cursor in the shell, and will likely be requested of the driver in the future

Reply:
Hmm, are you sure? The [documentation](https://docs.mongodb.com/manual/reference/method/cursor.allowDiskUse/) made it seem like it's only supported on find cursors:

> allowDiskUse() has the following form:
> `db.collection.find(<match>).sort(<sort>).allowDiskUse()`

Similarly the documentation for [aggregation cursors](https://docs.mongodb.com/manual/reference/method/db.collection.aggregate/#cursor-behavior) doesn't list that method. All the builder methods are [only listed](https://docs.mongodb.com/manual/reference/method/db.collection.find/#available-mongo-shell-cursor-methods) in the find cursor docs.

Reply:
That's strange, I'm just referring to what's supported on [the command](https://docs.mongodb.com/manual/reference/command/aggregate/#syntax). I assumed there would be builders for all of those properties, maybe they accidentally omitted it in the shell as part of the server work to implement that feature?

Reply:
I think it's worth deferring that work to a separate ticket, since it doesn't seem to be reference in the shell documentation currently. The [current guidance](https://docs.mongodb.com/manual/reference/command/aggregate/#aggregate-data-using-external-sort) is to use the `options` when calling `db.collection.aggregate(pipeline, options)` which we already support.